src/main/java/database/Storage.java

```diff
@@ -0,0 +1,16 @@
+package database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> assortment = new HashMap<>();
+
+    public static Map<String, Integer> getAssortment() {
+        return Map.copyOf(assortment);
+    }
+
+    public static void updateStorage(Map<String, Integer> transactions) {
```

Comment:
The `updateStorage` method is a violation of the Single Responsibility Principle (SRP). According to SRP, a class should have one and only one reason to change. The `Storage` class should only be responsible for storing data, not for updating it. The update logic should be placed in a separate service class.